content/developer/reference/frontend/services.rst

```diff
@@ -692,6 +694,58 @@ When a rpc fails, then:
   displayed and the server is regularly contacted until it responds. The
   notification is closed as soon as the server responds.
 
+.. _frontend/services/scroller:
+
+Scroller service
+----------------
+
+Overview
+~~~~~~~~
+
+- Technical name: `scroller`
+- Dependencies: none
+
+Provides a way to handle scrolls to anchors in the webclient. 
```

Comment:
maybe clearer to explain what it does. Suggestion: Replace the 2 lines with:

Whenever the user clicks on an anchor in the web client, this service automatically scrolls
to the target (if appropriate).